pymc3/distributions/discrete.py

```diff
@@ -154,13 +153,20 @@ def __init__(self, alpha, beta, n, *args, **kwargs):
 
     def _random(self, alpha, beta, n, size=None):
         size = size or 1
-        p = np.atleast_1d(stats.beta.rvs(a=alpha, b=beta, size=np.prod(size)))
+        p = stats.beta.rvs(a=alpha, b=beta, size=size).flatten()
         # Sometimes scipy.beta returns nan. Ugh.
         while np.any(np.isnan(p)):
             i = np.isnan(p)
             p[i] = stats.beta.rvs(a=alpha, b=beta, size=np.sum(i))
         # Sigh...
-        _n, _p, _size = np.atleast_1d(n).flatten(), p.flatten(), np.prod(size)
+        _n, _p, _size = np.atleast_1d(n).flatten(), p.flatten(), p.shape[0]
+
+        quotient, remainder = divmod(_p.shape[0], _n.shape[0])
```

Comment:
where is `divmod` defined? should it be np.divmod?

Reply:
oddly, it is a built-in!  so long as `_p` and `_n` are integers, this should be fine (and they are `flatten`ed just above, so this should be fine).